definitions/npm/newrelic_v4.x.x/test_newrelic_v4.x.x.js

```diff
@@ -0,0 +1,43 @@
+// @flow
```

Comment:
Put the test under the versioned flow directory.

Also, use describe/it blocks.

Reply:
Done